Makefile

```diff
@@ -0,0 +1,20 @@
+.PHONY: clean build test run
+
+BINARY_NAME=bin/scribe-server
+
+# Clean any build artifacts
```

Comment:
It does! Actually, Go has some pretty set out standards for comments (https://go.dev/doc/comment), since there is a built-in docs generation tool - I guess similar to docstrings in Python. 

This file here is just a Makefile - so murky standards for comments.. I can make full-sentence comments though to stick to Scribe-wide conventions :+1: